website/content/api-docs/features/consistency.mdx

```diff
@@ -2,20 +2,47 @@
 layout: api
 page_title: Consistency Modes
 description: >-
-  Most of the read query endpoints support multiple levels of consistency. Since
-  no policy will suit all clients' needs, these consistency modes allow the user
-  to have the ultimate say in how to balance the trade-offs inherent in a
-  distributed system.
+  Every Consul HTTP API read request uses one of several "consistency modes" that offer
+  different levels of response consistency and performance. By default, Consul uses a
+  consistency mode that is ideal for most use cases. Users can override consistency modes
+  to fine-tune these trade-offs for their own use case.
 ---
 
 # Consistency Modes
 
-Most of the read query endpoints support multiple levels of consistency. Since
-no policy will suit all clients' needs, these consistency modes allow the user
-to have the ultimate say in how to balance the trade-offs inherent in a
-distributed system.
+Every Consul HTTP API read request uses one of several "consistency modes" that offer
+different levels of response consistency and performance. By default, Consul uses a
+consistency mode that is ideal for most use cases. Users can override consistency modes
+to fine-tune these trade-offs for their own use case at two levels:
+- Per interface (HTTP API, DNS)
+- Per request (HTTP API requests only)
 
-The three read modes are:
+## What is Consistency?
```

Comment:
@trujillo-adam : I've added this section to introduce the concept of consistency